jsoc/index.md

```diff
@@ -26,6 +26,13 @@ Those participating in the program can check out [the getting started page](/lea
 Are you interested in Julia but not super familiar with the language?
 Head over to [JuliaAcademy](https://juliaacademy.com) to take some free online Julia classes.
 
+## Google Season of Docs (GSoD)
+
+The Julia organization is applying to participant in [Google's Season of Docs](https://developers.google.com/season-of-docs).
+The Julia organization generally fields a large number of students mentored by open source developers, university professors at leading institutions from all around the world, and technical writers in the Julia Community. 
+Students who are accepted into this prestigious program are paid a stipend to contribute open source documentation around  around Julia and its package ecosystem.
```

Comment:
around  2x, contribute _to_ ? (maybe not)

Reply:
Fixed this, thanks for pointing it out